src/test/java/org/apache/commons/crypto/stream/AbstractCipherStreamTest.java

```diff
@@ -313,7 +313,6 @@ protected CryptoCipher getCipher(String cipherClass) throws IOException {
 
     @Test
     public void testReadWrite() throws Exception {
-        Assert.assertEquals(null, Openssl.getLoadingFailureReason());
```

Comment:
If native code is failing to load, the exception will be caught by throwable block. This is designed for fall back mechanism for cipher loader.